main.tf

```diff
@@ -34,6 +34,13 @@ resource "aws_ec2_transit_gateway" "this" {
   auto_accept_shared_attachments  = var.enable_auto_accept_shared_attachments ? "enable" : "disable"
   vpn_ecmp_support                = var.enable_vpn_ecmp_support ? "enable" : "disable"
   dns_support                     = var.enable_dns_support ? "enable" : "disable"
+  transit_gateway_cidr_blocks     = var.transit_gateway_cidr_blocks
+
+  timeouts {
+    create = try(var.timeouts.create, null)
```

Comment:
Just wondering if we should write vars like this or `var.timeouts["create"]`? What is more correct? I think we write it like `var.timeouts["create"]` in most places. Are there any drawbacks/benefits of any? I am lost :)

Reply:
I don't think there are any drawbacks of one way vs the other - this way was just less characters and fit with the other `try()` usage patterns